phoenix-core/src/main/java/org/apache/phoenix/mapreduce/index/IndexUpgradeTool.java

```diff
@@ -707,12 +716,12 @@ private int startIndexRebuilds(HashMap<String, IndexInfo> indexInfos,
             list.add("-tenant");
             list.add(tenantId);
         }
-        if (syncRebuild) {
-            list.add("-runfg");
-        }
-        if(!Strings.isNullOrEmpty(verify)) {
-            list.add("-v");
-            list.add(verify);
+
+        if(!Strings.isNullOrEmpty(indexToolOpts)) {
+            String[] options = indexToolOpts.split("\\s+");
```

Comment:
how would this work in a bash script? Does bash identify consecutive options like this?
-tool -v ONLY

Reply:
should be fine, but good to double-check?

Reply:
They will have to be quoted. Like -tool '-v ONLY -runfg -starttime 10000'

Reply:
I double checked by running the tool on the shell